validator/client/validator_propose_test.go

```diff
@@ -113,7 +113,7 @@ func TestProposeBlock_UsePendingDeposits(t *testing.T) {
 
 	m.proposerClient.EXPECT().PendingAttestations(
 		gomock.Any(), // ctx
-		gomock.Eq(&ptypes.Empty{}),
+		gomock.Any(),
```

Comment:
Why changing all of these? If the type has changed, can you use the proper type now?